src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -751,4 +705,34 @@ private void assertActionCommentIsAbsentInLoan(JsonObject loan) {
     assertThat("'actionComment' property should be absent in loan",
       loan.getString(ACTION_COMMENT_KEY), nullValue());
   }
+
+  private void verifyRenewedLoan(IndividualResource smallAngryPlanet,
+                                 IndividualResource jessica, JsonObject renewedLoan) {
```

Comment:
Please use a two space indentation for continuation rather than horizontal alignment, that is difficult to maintain and wastes horizontal space

Reply:
Done